### PR TITLE
Adds support for https://exchangerate.host/ as a driver

### PR DESCRIPTION
**Pull request by lukeraymonddowning**
See https://github.com/worksome/exchange/discussions/3 for more details

### CODE REVIEW

**Comment by lukeraymonddowning** [comment]
Also wondering if we should make this the default seen as it requires no authentication. That would mean a user installing the package would be up and running practically instantly.

File: src/ExchangeRateProviders/ExchangeRateHostProvider.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+use Worksome\Exchange\Contracts\ExchangeRateProvider;
+use Worksome\Exchange\Support\Rates;
+
+final class ExchangeRateHostProvider implements ExchangeRateProvider
+{
+    private FixerProvider $fixerProvider;
+
+    public function __construct(private Factory $client)
+    {
+        $this->fixerProvider = new FixerProvider($this->client, '', 'https://api.exchangerate.host');
```
Comment (olivernybroe) [inline src/ExchangeRateProviders/ExchangeRateHostProvider.php:17]: If we get a third implementation that has the same API, then let's move the current fixer into an abstract class instead, but for now let's just do it with a decorator

**Review by olivernybroe — APPROVED** [review]
Looks good, haven't tested it locally, but I trust you 👍

**Comment by olivernybroe** [comment]
@lukeraymonddowning yeah, let's make it the default 👍 seems like a better user experience

**Comment by leasify-it** [comment]
Exactly the same API but it's free and the response from their serveris is faster.
Great work! Thanks!

**Pull request merged**